extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -479,7 +481,8 @@ export class AmpSelector extends AMP.BaseElement {
     }
 
     this.setInputs_();
-    this.fireSelectEvent_(el);
+    // "selectUp" and "selectDown" actions only require "low" trust.
+    this.fireSelectEvent_(el, ActionTrust.LOW);
```

Comment:
Same comment as above, this should propagate the trust, rather than just firing low trust.